test/unit/add_annotation.spec.js

```diff
@@ -4,8 +4,8 @@ import { mountWithIntl } from './helpers/intl-test';
 import AddAnnotation from '../../src/app/components/annotations/AddAnnotation';
 
 describe('<AddAnnotation />', () => {
-  const annotated_archived = { archived: true };
-  const annotated_not_archived = { archived: false };
+  const annotated_archived = { archived: 1 };
+  const annotated_not_archived = { archived: 0 };
```

Comment:
Is there a reason for not importing `CheckArchivedFlags` and using the constants from the tests too?

Reply:
Fixed by using CheckArchivedFlags constants